src/Connection.cpp

```diff
@@ -227,12 +227,15 @@ quint16 Connection::localPort() const {
 	return qtsSocket->localPort();
 }
 
+// Returns the peer's chain of digital certificates, starting with the peer's immediate certificate
+// and ending with the CA's certificate.
```

Comment:
I guess technically function documentation should go into the header file, where the function is declared. And there it should also use Doxygen-style comments (`///` instead of `/`) :point_up: